bodhi/tests/server/functional/test_generic.py

```diff
@@ -697,3 +711,13 @@ def test_home_counts(self):
         </a>
           <span class="label label-default">prerelease</span>
       </h3>""", res)
+
+
+class TestNotfoundView(base.BaseTestCase):
+    """Test the notfound_view() handler."""
+    def test_notfound(self):
+        """Assert that we correctly deal with 404's."""
+        res = self.app.get('/makemerich', status=404)
```

Comment:
Well tried :) but you won't fool Bodhi that easily!

Reply:
@pypingou hehe